Updating Social Media Guidelines (#4680)
kubernetes/community: communication/marketing-team/social-guidelines.md

# Social media guidelines

**Twitter:** [K8sContributor](https://twitter.com/k8scontributors)

## General guidelines

All messaging must be consistent with the values and principles of the project and the [ethos/vision of the team](https://github.com/kubernetes/community/blob/master/communication/marketing-team/CHARTER.md#ethosvision). Messaging should be positive and uplifting. Be aware that sarcasm and/or jokes are generally hard to read over a medium such as social media.

> Original messages should come out of this teams' accounts at least twice a week (if not more). However, there should be no more than three original messages per day. Retweets, likes, and responses are unlimited but, they should be thoughtful. All event messages should have an action and/or thumbnail image (click link for details, register today, etc. are example actions). Never include photos, user handles, or other personally identifiable information without explicit permission.

Reviewer: Minor recommendation -- the only limit I find worth setting explicitly is _tweets per hour_. Said another way, if we have enough to share where we can add something worthwhile every hour in the day, that'd be wonderful. If you agree, here's a suggestion: 

```suggestion
Original messages should come out of this teams' accounts at least twice a week (if not more). However, there should be no more than three tweets per hour (including retweets). Retweets, likes, and responses are unlimited, but they should be used thoughtfully to encourage inclusive and kind participation. Use scheduling software as needed to space out messages. All messages should serve a purpose: have an action and/or thumbnail image ("click link for details," "register today," etc. are example actions). Never include photos, user handles, or other personally identifiable information without explicit permission.
```

Attempt to keep abbreviations to a minimum. Things like `k/k`, for example, might be known in the community, but for new contributors, this is something that is learned with time. Minimizing abbreviations and using plain language is strongly encouraged.

Responses should include helpful links, validation, and/or additional information. Responses should occur within 48 hours. At the very least, like things as an acknowledgment. Sources of information should include Kubernetes contributor related web properties and documentation. TODO: CLEARLY DEFINE THOSE WEB PROPERTIES

> Ideally, no individuals outside of the contributor community with the possible exception that they are talking about contributors should be an considered a topic of messaging. Example: Bill Gates isn't part of the Kubernetes community, it wouldn't be prudent to message him.

Reviewer: I found the last part could be more explicit for clarity: 
```suggestion
Ideally, no individuals outside of the contributor community with the possible exception that they are talking about contributors should be an considered a topic of messaging. Example: Bill Gates isn't part of the Kubernetes community, so tweets with @BillGates will not be published.
```

Other CNCF projects, unless there is a common contributor cause, are out of scope for coverage by this team's social media accounts.

Kubernetes Organization Members with access to account are:
- @parispittman
- @mrbobbytables
- @chris-short

## Themes/Topics

> - Contributor/SIG blog posts
> - Events
> - KEPs
> - Process changes
> - Release info, code freeze, et al
> - Significant historical events as related to the project (Kubernetes birthday, previous events, etc.)

Reviewer: ```suggestion
- Contributor/SIG blog posts
- Events
- KEPs
- Process changes
- Release info, code freeze, et al
- Accomplishments (reaching 100,000 GitHub stars for example)
- Significant historical events as related to the project (Kubernetes birthday, previous events, etc.)
```

> ### Open-ended questions

Reviewer: Is this an FAQ? Let's call it that. 

```suggestion
### Frequently Asked Questions (FAQ)
```

> This team's social media accounts need to be incredibly mindful of who it follows. As it is an information dissemination channel some might think that following no one is a better choice but, that could end up limiting exposure. Attempt to keep follower count low to make it easier to manage.

Reviewer: This section could use a crisper answer. The two strategies I like for information accounts are to either (a) Follow no one: write only mode or (b) Follow everybody: get great visibility by people seeing it in their follows and following back. 

Which makes sense to you @chris-short? 

```suggestion
Q: Will the account follow other accounts?
A: TODO (Y/N). The account is designed for information dissemination, so we will focus on that by (following no one and focusing on sharing || following everyone to extend our reach).
```

> Trolls: Don't feed them, mute them. Block them as a last resort and only after discussing with team members. Reasons to block include verbal abuse, violating the platform's terms, or violating the [Code of Conduct](https://github.com/kubernetes/community/blob/master/code-of-conduct.md). The community is open and its accounts should be too but, there are limits.

Reviewer: ```suggestion
Q: Do we engage with hostile messages? 
A: No. Don't feed the trolls. Mute them is the correct response. Block them as a last resort and only after discussing with team members. Reasons to block include verbal abuse, violating the platform's terms, or violating the [Code of Conduct](https://github.com/kubernetes/community/blob/master/code-of-conduct.md). The community is focused on openness and our accounts should be too, but there are times when we should reduce harmful people's visibility through blocking.
```
